src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyDiscoveryManager.cs

```diff
@@ -102,6 +102,7 @@ public class ProxyDiscoveryManager : IProxyDiscoveryManager, IBaseProxy, ITestDi
         IRequestData requestData,
         ITestRequestSender requestSender,
         ITestRuntimeProvider testHostManager,
+        Framework testhostManagerFramework,
```

Comment:
Would have put it on the teshost manager, but I have no idea what type it is, maybe I should wrap it in a container type. But this is also okay.